### DISCUSSION TITLE
Welcome to TechnoTrek Discussions!

### DISCUSSION THREAD

**Discussion by smghazijaffri** [smghazijaffri/TechnoTrek#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Help make this better by demonstrating your skills and capabilities
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SaadRehman123** [comment]
What are you doing? Boi

**Reply by smghazijaffri** [reply]
Requesting help from the community
